### PR TITLE
Simple Combobox support in thmutil

### PR DESCRIPTION
**Pull request by Mike-GC**
Simple combobox support in thmutil. Does not yet support populating items in the combobox today from XML, you must insert the items programmatically.

### CODE REVIEW

File: src/libs/dutil/thmutil.cpp [hunk 1]
```diff
@@ -2331,6 +2336,11 @@ static HRESULT ParseControls(
         {
             type = THEME_CONTROL_TYPE_TAB;
         }
+        else if (CSTR_EQUAL == ::CompareStringW(LOCALE_INVARIANT, 0, bstrType, -1, L"Combobox", -1) ||
+                 CSTR_EQUAL == ::CompareStringW(LOCALE_INVARIANT, 0, bstrType, -1, L"cm", 2))
```
Comment (barnson) [inline src/libs/dutil/thmutil.cpp]: Any reason to introduce a new old-school name? They're all obsolete in WiX v4.0.

File: src/libs/dutil/thmutil.cpp [hunk 2]
```diff
@@ -2331,6 +2336,11 @@ static HRESULT ParseControls(
...
+        {
+            type = THEME_CONTROL_TYPE_COMBOBOX;
+        }
 
         if (THEME_CONTROL_TYPE_UNKNOWN != type)
         {
```
Comment (barnson) [inline src/libs/dutil/thmutil.cpp:2345]: Is everyone supposed to use ThemeSendControlMessage/ThemePostControlMessage?

File: src/libs/dutil/thmutil.cpp (hunk 1 above)
Comment (Mike-GC) [inline src/libs/dutil/thmutil.cpp]: Good point, I'll remove it. Schema doesn't support them anyway.

File: src/libs/dutil/thmutil.cpp (hunk 2 above)
Comment (Mike-GC) [inline src/libs/dutil/thmutil.cpp:2345]: Yes, I started writing a wrapper to insert strings and get the current selection, and I realized it didn't have anything interesting to do, the messages CB_ADDSTRING and CB_GETCURSEL are just extremely simple. If you feel I should add wrapper functions for these (that are probably going to be oneliners), speak up, I don't feel strongly about it.

Comment (barnson) [inline src/libs/dutil/thmutil.cpp:2345]: Nope, that's cool.

**Pull request merged**